src/jaxsim/rbda/actuation/common.py

```diff
@@ -0,0 +1,12 @@
+import jaxsim.typing as jtp
+from jaxsim.utils import JaxsimDataclass
+
+
+class ActuationParams(JaxsimDataclass):
+    """
+    Parameters class for the actuation model.
+    """
+
+    τ_max: jtp.Float = 3000.0  # Max torque (Nm)
+    ω_th: jtp.Float = 30.0  # Threshold speed (rad/s)
+    ω_max: jtp.Float = 100.0  # Max speed for torque drop-off (rad/s)
```

Comment:
Can we use `dataclasses.field` as these attributes are type-annotated and so that they appear in `__repr__()`?
```suggestion
    τ_max: jtp.Float = dataclasses.field(default=3000.0)  # Max torque (Nm)
    ω_th: jtp.Float = dataclasses.field(default=30.0)  # Threshold speed (rad/s)
    ω_max: jtp.Float = dataclasses.field(
        default=100.0
    )  # Max speed for torque drop-off (rad/s)
```

Reply:
done in ba556cd4e97d43b10710ad06c9b3af842ddbcd8b

Reply:
This actually  does not make the field instantiate correctly  see 
     
     ```
     """
        Compute the torque limits using the tn curve.
    
        Args:
            model: The model to consider.
            data: The data of the considered model.
    
        Returns:
            The torque limits.
        """
    
        τ_max = model.actuation_params.torque_max  # Max torque (Nm)
        ω_th = model.actuation_params.omega_th  # Threshold speed (rad/s)
        ω_max = model.actuation_params.omega_max  # Max speed for torque drop-off (rad/s)
        abs_vel = jnp.abs(data.joint_velocities)
        τ_lim = jnp.where(
>           abs_vel <= ω_th,
            τ_max,
            jnp.where(
                abs_vel <= ω_max, τ_max * (1 - (abs_vel - ω_th) / (ω_max - ω_th)), 0.0
            ),
        )
E       TypeError: '<=' not supported between instances of 'DynamicJaxprTracer' and 'Field'
E       --------------------
E       For simplicity, JAX has removed its internal frames from the traceback of the following exception. Set JAX_TRACEBACK_FILTERING=off to include these.

```

Do you know why this happens with fields ?

Reply:
Sorry, I haven't seen this before writing https://github.com/ami-iit/jaxsim/pull/405#issuecomment-2797047047

Reply:
Should be done now !